rmake.py

```diff
@@ -118,7 +118,7 @@ def parse_args():
                         help='[DEPRECATED] Linux only: Build hipBLAS using hipcc compiler. Deprecated, use --compiler instead.')
 
     parser.add_argument(      '--no-hip-clang', dest='use_hipcc_compiler', required=False, default=True, action='store_false',
-                        help='[DEPRECATED] Linux only: Build hipBLAS with g++ compiler instead of hipcc compiler. Deprecated, use --compiler instead.')
+                        help='[DEPRECATED] Linux only: Build hipBLAS with default amdclang++ compiler instead of hipcc compiler. Deprecated, use --compiler instead.')
```

Comment:
I wouldn't change a deprecated flag, it was to use g++ so still valid  as is (hipcc and amdclang are the same compiler)